fe/fe-core/src/main/java/org/apache/doris/httpv2/HttpServer.java

```diff
@@ -93,4 +101,16 @@ public void start(String dorisHome) {
                 .properties(properties)
                 .run(new String[]{});
     }
+
+    @Bean
+    public Jackson2ObjectMapperBuilderCustomizer jacksonObjectMapperCustomization() {
```

Comment:
This may affect all HTTP api, I am not sure if this is ok for result of other APIs

Reply:
1. I tested it and there is no problem
2. Because of the potential overflow problem of json numbers, these types of return values should be converted into strings